thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallService.java

```diff
@@ -63,38 +57,20 @@ public void onError(Exception e) {
     };
 
     /**
-     * Creates a new {@link ThriftCallService} with the specified service implementation.
-     *
-     * @param implementation an implementation of {@code *.Iface} or {@code *.AsyncIface} service interface
-     *                       generated by the Apache Thrift compiler
+     * Creates a new instance of {@link ThriftCallServiceBuilder} which can build
+     * an instance of {@link ThriftCallService} fluently.
      */
-    public static ThriftCallService of(Object implementation) {
-        requireNonNull(implementation, "implementation");
-        return new ThriftCallService(ImmutableMap.of("", ImmutableList.of(implementation)));
-    }
-
-    /**
-     * Creates a new multiplexed {@link ThriftCallService} with the specified list service implementations.
-     *
-     * @param implementations a {@link Map} whose key is service name and value is the list of implementations
-     *                        of {@code *.Iface} or {@code *.AsyncIface} service interface generated by
-     *                        the Apache Thrift compiler
-     */
-    public static ThriftCallService of(Map<String, ? extends Iterable<?>> implementations) {
-        requireNonNull(implementations, "implementations");
-        return new ThriftCallService(implementations);
+    public static ThriftCallServiceBuilder builder() {
+        return new ThriftCallServiceBuilder();
     }
```

Comment:
Ditto, we need this method so we don't break backward compatibility.

Reply:
I added it. Thank you!

Reply:
It looks like it's not added back? There's no `public static ThriftCallService of(Map<String, ? extends Iterable<?>> implementations)` in `ThriftCallService` anymore.

Reply:
I added it again. thank you!